docs/content/en/docs/reference/crd-reference/analysisdefinition.md

```diff
@@ -132,21 +139,21 @@ spec:
       target:
         failure:
           <operator>:
-            fixedValue: integer> |
+            fixedValue: <quantity> |
             inRange: | notInRange:
               lowBound: <integer-quantity>
               highBound: <integer-quantity>
         warning:
           <operator>:
-            fixedValue: integer> |
+            fixedValue: <quantity> |
```

Comment:
to stay consistent with what we have for the `low/highbound` properties, let's write it like this

```suggestion
            fixedValue: <integer-quantity> |
```